Functions/InputOutput/xASL_io_dcm2nii.m

```diff
@@ -73,9 +73,6 @@
     msg = [];
 
     %% 3. Locate dcm2nii executable
-	%if regexpi(inpath,'PAR$') % Automatic detection of PAR-REC, now disabled
-	%	imPar.dcm2nii_version = '20101105';
-	%end
 	
 	if ismac && str2num(imPar.dcm2nii_version(1:4))<2014

```

Comment:
I guess this can move out, we don't have any dcm2nii_version before 2014 anymore :)
And macOS is fine with the versions that we have now

Reply:
removed 06d28576